wai-aria/tools/convert_wiki.pl

```diff
@@ -0,0 +1,648 @@
+#!/usr/bin/perl
+#
+#  convert_wiki.pl - Transform an old-style wiki into the new format
+#
+#  This script assumes that a wiki has testable statement entries
+#  with varying lemgth lines. Those lines will be converted into
+#  the format described by the specification at
+#  https://spec-ops.github.io/atta-api/index.html
+#
+#  usage: convert_wiki.pl -f file | -w wiki_title -o outFile
+
+use strict;
+
+use IO::String ;
+use JSON ;
+use MediaWiki::API ;
+use Getopt::Long;
+
+my @apiNames = qw(UIA MSAA ATK IAccessible2 AXAPI);
+
+# dir is determined based upon the short name of the spec and is defined
+# by the input or on the command line
+
+my $file = undef ;
+my $spec = undef ;
+my $wiki_title = undef ;
+my $dir = undef;
+my $outFile = undef;
+
+my $result = GetOptions(
+    "f|file=s"   => \$file,
+    "w|wiki=s"   => \$wiki_title,
+    "s|spec=s"   => \$spec,
+    "o|output=s"   => \$outFile);
+
+my $wiki_config = {
+  "api_url" => "https://www.w3.org/wiki/api.php"
+};
+
+my %specs = (
+    "aria11" => {
+      title => "ARIA_1.1_Testable_Statements",
+      specURL => "https://www.w3.org/TR/wai-aria11"
+    },
+    "svg" => {
+      title => "SVG_Accessibility/Testing/Test_Assertions_with_Tables_for_ATTA",
+      specURL => "https://www.w3.org/TR/svg-aam-1.0/"
+    }
```

Comment:
My understanding is that SVG ARIA / AAM is on hold. And last time I checked, the tables in the wiki at the title above were not runnable. On the flip side, DPB AAM testable statements are done (by me) and work as expected at least for Linux and OS X. What do you think about nuking SVG and adding DPUB here?